src/SocialServer/Network/Send.cs

```diff
@@ -0,0 +1,225 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Melia.Shared.Network;
+using Melia.Social.Database;
+
+namespace Melia.Social.Network
+{
+	public static class Send
+	{
+		/// <summary>
+		/// Login succeeded
+		/// </summary>
+		/// <param name="conn"></param>
+		public static void SC_LOGIN_OK(ISocialConnection conn)
+		{
+			var packet = new Packet(Op.SC_LOGIN_OK);
+
+			conn.Send(packet);
+		}
+
+		/// <summary>
+		/// Dummy
+		/// </summary>
+		public static void SC_FROM_INTEGRATE(ISocialConnection conn, int type)
+		{
+			var packet = new Packet(Op.SC_FROM_INTEGRATE);
+
+			switch (type)
+			{
+				case 0x01:
+					packet.PutLong(conn.Account.Id);
+					packet.PutInt(0);
+					break;
+				case 0x14:
+					packet.PutLong(conn.Account.Id);
+					packet.PutInt(0); // 728
+					break;
+			}
+			conn.Send(packet);
+		}
+
+		/// <summary>
+		/// Login related
```

Comment:
More elaborate comments might help us all with understanding and categorizing code in the future^^ "Login related" is a start, but is it about? When is it sent? What does it do? What happens if it's not sent?

Reply:
Excellent question, I usually don't test what happens if I don't send it but I'll take a look, to see what the behavior is.